src/stars.css

```diff
@@ -0,0 +1,24 @@
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background: url(./images/star.svg) no-repeat;
+  background-position: center center;
```

Comment:
You can write 
background: url(./images/star.svg) no-repeat center ;

Reply:
> You can write background: url(./images/star.svg) no-repeat center ;

True, less code is better.
Thanks)